template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/validation/SafetyValveValidator.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.cmtemplate.validation;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.sequenceiq.cloudbreak.template.model.SafetyValve;
+
+@Component
+public class SafetyValveValidator {
+    public static final List<String> NON_XML_FORMAT_SAFETY_VALVE = List.of("hue_service_safety_valve",
```

Comment:
Please also add `hue_server_hue_safety_valve`, it is a Safety Valve for role `HUE_SERVER`.

Reply:
I did not find it in any template thats why i did not add it. Can you point me where I can see the value for hue_server_hue_safety_valve

Reply:
Why does it matter if no blueprints use it at the moment? It is better to be prepared, and `hue_server_hue_safety_valve` may get used in future built-in blueprints.

You can actually see a few examples in test blueprints under `core/src/test/resources/test/defaults/blueprints` where `hue_server_hue_safety_valve` is also used, even though they will surely not be validated here, :smile:

Reply:
I have added hue_server_hue_safety_valve.